### ISSUE TITLE
This code is incomplete

### ISSUE THREAD

**Issue by Noemata** [tgckpg/playground#1]
This looks like a partial submission.  Looked at your blog entry related to this.  What you describe there appears to be a working implementation.

**Comment by tgckpg** [comment]
whoa, didn't expect a surprise. Did you mean the perceptron impl and the pattern recognition I've mentioned on my blog?

Welp, since this repo is my **playground**. I think the drawing impl is buried somewhere in the commit history.

Anyway, the perceptron is just an experiment to try to solve a problem I've created to solve another problem I've created to solve ... ( repeat this about 5 times ) but then I realized I have to solve another issue before solving this issue. But then I realized ... ( repeat this about 7 times ).

SO currently I am in the CodePage Encoding universe scratching my heads off about converting a mixed encoding for this fabulous EBXA-C formatted dictionary standard.

Oh right, about this perceptron thing, I couldn't promise anything but I think I'll get back to it eventually.

**Comment by Noemata** [comment]
Yes, that is correct, I am referring to the code associated with your blog.  I don't expect you to dust this off if you have gone through a series of iterations that have left it in this state.  As you stated, it is your playground.  Initially I thought the code was an incomplete push that needed another submission.  Leave it be @tgckpg.

Thank you for the feedback.
